tests/viewsets/test_locations.py

```diff
@@ -54,6 +55,33 @@ def test_create_location(self):
         """
         self._create_location()
 
+    def test_create_location_with_shapefile(self):
+        """
+        Test that we can create a Location Object with a shapefile
+        """
+        user = mommy.make('auth.User')
+        # Import Shapefile
+        ds = DataSource('tests/fixtures/test_shapefile.shp')
+        lyr = ds[0]
+        # Transform First Data Entry into wkt format
+        item_wkt = lyr[1].geom.wkt
```

Comment:
I think we need to change out approach.  We want to imagine ourselves in the position of  someone who just wants to upload shapefile.  We cannot and should not assume that they can do this fancy footwork of extracting this from the .shp file

Reply:
This might help, although you are doing the same thing: https://github.com/onaio/mspray/blob/master/mspray/apps/main/management/commands/load_locations_shape_file.py